src/js/index.js

```diff
@@ -52,6 +52,17 @@ import { VTT_MODULE_NAME } from "./settings/constants";
     return result;
   }
 
+  function addCustomItemsToGameMenu(wrapper) {
+    const items = wrapper();
+    items['close'] = {
+      label: "MENU.Close",
+      icon: '<i class="fas fa-times"></i>',
+      enabled: true,
+      onClick: () => ui.menu.close()
```

Comment:
Adds a bit more usability to the game menu by allowing to close it with a menu entry

Reply:
jup good idea!